nsls2forge_utils/cli.py

```diff
@@ -49,5 +50,45 @@ def main():
         check_package_version(**version_kwargs)
 
 
-if __name__ == '__main__':
-    sys.exit(main())
+def all_feedstocks():
+    parser = argparse.ArgumentParser(
+        description='Retrieve all feedstock repositories from cache or GitHub '
+                    'and write to file.')
```

Comment:
Does this multiline splitting work fine? I thought we'd need the parentheses around the text for `description`.

Reply:
Yes, it works on my machine. I'll add parentheses anyway just in case older python versions don't support it.